axum-extra/Cargo.toml

```diff
@@ -102,4 +104,8 @@ allowed = [
     "tokio",
     "tower_layer",
     "tower_service",
+    # typed-json does not appear in the public API, but does appear in `#[doc(hidden)]` re-exports
+    # accessed by macros that are not a part of the public API.
+    # See also: https://github.com/davidpdrsn/cargo-public-api-crates/issues/6
+    "typed-json",
```

Comment:
AFAIK, `cargo-public-api-crates` operates on rustdoc JSON, so I wonder why this is necessary.

Reply:
Huh. I might be wrong about this. I need to do some more investigation.